packages/zero-protocol/src/row-patch.ts

```diff
@@ -20,7 +20,14 @@ const updateOpSchema = v.object({
 const delOpSchema = v.object({
   op: v.literal('del'),
   tableName: v.string(),
-  id: primaryKeyValueRecordSchema,
+  // Either `id` or `value` must be set.
```

Comment:
Right now id always need to be set until MIN_SUPPORTED_PROTOCOL_VERSION is bumped to 4 right?